src/main/java/seedu/address/model/event/Event.java

```diff
@@ -220,18 +227,18 @@ public boolean equals(Object other) {
     public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
         return Objects.hash(name, venue, manpowerNeeded,
-                startDate, endDate, manpowerAllocatedList, tags);
+                startDate, endDate, manpowerAllocatedList, eventDateTimeMap, tags);
     }
 
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
         builder.append(getName());
-        builder.append(" Event Venue: ").append(getVenue());
-        builder.append(" Event Manpower Needed: ").append(getManpowerNeeded());
-        builder.append(" Event Start Date: ").append(getStartDate());
-        builder.append(" Event End Date: ").append(getEndDate());
-        builder.append(" Tags: ");
+        builder.append("\nEvent Venue: ").append(getVenue());
```

Comment:
why add \n? I think it will affect output during add,edit? Theres another tostring variation?

Reply:
It will display each field on a new line which makes it much more user-friendly and readable.